README.md

```
@@ -1,102 +1,176 @@
-}
- ```
-
-After filling the config, you can run External server with this command:
+# Unit tests
+
+## Necessary steps before testing
+
+### Requirements
+
+- [Requirements](#requirements)
+- CMLIB: https://github.com/cmakelib/cmakelib
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)